README.md

```diff
@@ -81,13 +81,29 @@ releases of the toolkit.
     * GATK4 uses the [Conda](https://conda.io/docs/index.html) package manager to establish and manage the
       Python environment and dependencies required by GATK tools that have a Python dependency. This environment also 
       includes the R dependencies used for plotting in some of the tools. The ```gatk``` environment 
-      requires hardware with AVX support for tools that depend on TensorFlow (e.g. CNNScoreVariant). The GATK Docker image 
+      requires hardware with AVX support for tools that depend on TensorFlow. The GATK Docker image 
       comes with the ```gatk``` environment pre-configured.
-      	* At this time, the only supported platforms are 64-bit Linux distributions. The required Conda environment is not
-	  currently supported on OS X/macOS. 
     * To establish the environment when not using the Docker image, a conda environment must first be "created", and
       then "activated":
-        * First, make sure [Miniconda or Conda](https://conda.io/docs/index.html) is installed (Miniconda is sufficient).
+        * First, make sure [Miniconda or Conda](https://conda.io/docs/index.html) is installed. We recommend installing
```

Comment:
Should we add a parenthetical note saying that you might run into issues if you try to use a different conda version?

Reply:
Yes, we definitely should!

Reply:
What is the opt folder and why do we install there?  I've never understood linux folder structures...

Reply:
According to one of my AI friends:

The /opt directory in Linux is a reserved space for installing optional or add-on software packages. The name /opt stands for "optional". 

The /opt directory is important for several reasons, including: 

* System management
    * The /opt directory helps keep the Linux system organized and modular, which makes it easier to manage and maintain. 
* Security
    * By separating optional software from the core system, the /opt directory reduces the risk of accidentally modifying or deleting critical system files. 
* Software storage
    * **The /opt directory is often used to store manually compiled software, which is built from source code instead of being installed from distribution repositories.** 
    
Within the /opt directory, applications are usually stored in /opt/bin, while libraries are placed in /opt/lib